clueless/settings.py

```diff
@@ -118,3 +117,5 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+from .settings_secret import *
```

Comment:
Odd, I see that on my phone but I swear I didn't see that last night on my desktop.  Let me see if I can clear that out